usage/local-api.md

```diff
@@ -30,7 +30,7 @@ The Local API is one method of obtaining data from YouTube. For another method o
 | ---------------------------------------------------------------------------------------- | ------- | ----------------------------------------------------------------------------------------------------------------- | -------------------------------- |
 | [youtubei.js](https://github.com/LuanRT/YouTube.js)                                      | MIT     | Search Functionality + Suggestions, Playlists, Trending, Obtain Video Information, Generate DASH Files, Live Chat | No                               |
```

Comment:
Forgot to add Comments to this line 😆

Reply:
Do you mean adding fetching comments to yt.js?

Reply:
Yeah, I was mostly joking though because we'll probably make that line a lot smaller once we migrate from yt-channel-info

Reply:
Haha okay i initially thought that u were joking but then my head said but maybe it isnt a joke?

So yeah my brain playing tricks on me😅

Should a create a pr for adding the comments?

Reply:
I dont think it's necessary 🙂